<launch>
  <include file="$(find pr2_joint_position_controllers)/launch/pr2_controller.launch"/>
  <node pkg="pr2_cartesian_clients" type="manipulation_client" name="manipulation_client"  output="screen">
    <rosparam file="$(find pr2_cartesian_clients)/config/client_config.yaml" command="load"/>
  </node>

</launch>
